DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getMe HTTP/1.1" 200 202
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getMe HTTP/1.1" 200 202
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=1&timeout=20 HTTP/1.1" 200 1190
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): api.telegram.org:443
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "POST /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/editMessageText?text=6&chat_id=68978487&message_id=17&reply_markup=%7B%22inline_keyboard%22%3A+%5B%5B%7B%22text%22%3A+%22%22%2C+%22callback_data%22%3A+%22no%22%7D%2C+%7B%22text%22%3A+%22C%22%2C+%22callback_data%22%3A+%22C%22%7D%2C+%7B%22text%22%3A+%22%3C%3D%22%2C+%22callback_data%22%3A+%22%3C%3D%22%7D%2C+%7B%22text%22%3A+%22%2F%22%2C+%22callback_data%22%3A+%22%2F%22%7D%5D%2C+%5B%7B%22text%22%3A+%227%22%2C+%22callback_data%22%3A+%227%22%7D%2C+%7B%22text%22%3A+%228%22%2C+%22callback_data%22%3A+%228%22%7D%2C+%7B%22text%22%3A+%229%22%2C+%22callback_data%22%3A+%229%22%7D%2C+%7B%22text%22%3A+%22%2A%22%2C+%22callback_data%22%3A+%22%2A%22%7D%5D%2C+%5B%7B%22text%22%3A+%224%22%2C+%22callback_data%22%3A+%224%22%7D%2C+%7B%22text%22%3A+%225%22%2C+%22callback_data%22%3A+%225%22%7D%2C+%7B%22text%22%3A+%226%22%2C+%22callback_data%22%3A+%226%22%7D%2C+%7B%22text%22%3A+%22-%22%2C+%22callback_data%22%3A+%22-%22%7D%5D%2C+%5B%7B%22text%22%3A+%221%22%2C+%22callback_data%22%3A+%221%22%7D%2C+%7B%22text%22%3A+%222%22%2C+%22callback_data%22%3A+%222%22%7D%2C+%7B%22text%22%3A+%223%22%2C+%22callback_data%22%3A+%223%22%7D%2C+%7B%22text%22%3A+%22%2B%22%2C+%22callback_data%22%3A+%22%2B%22%7D%5D%2C+%5B%7B%22text%22%3A+%22%22%2C+%22callback_data%22%3A+%22no%22%7D%2C+%7B%22text%22%3A+%220%22%2C+%22callback_data%22%3A+%220%22%7D%2C+%7B%22text%22%3A+%22%2C%22%2C+%22callback_data%22%3A+%22.%22%7D%2C+%7B%22text%22%3A+%22%3D%22%2C+%22callback_data%22%3A+%22%3D%22%7D%5D%5D%7D HTTP/1.1" 200 935
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394660&timeout=20 HTTP/1.1" 200 1189
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "POST /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/editMessageText?text=6-&chat_id=68978487&message_id=17&reply_markup=%7B%22inline_keyboard%22%3A+%5B%5B%7B%22text%22%3A+%22%22%2C+%22callback_data%22%3A+%22no%22%7D%2C+%7B%22text%22%3A+%22C%22%2C+%22callback_data%22%3A+%22C%22%7D%2C+%7B%22text%22%3A+%22%3C%3D%22%2C+%22callback_data%22%3A+%22%3C%3D%22%7D%2C+%7B%22text%22%3A+%22%2F%22%2C+%22callback_data%22%3A+%22%2F%22%7D%5D%2C+%5B%7B%22text%22%3A+%227%22%2C+%22callback_data%22%3A+%227%22%7D%2C+%7B%22text%22%3A+%228%22%2C+%22callback_data%22%3A+%228%22%7D%2C+%7B%22text%22%3A+%229%22%2C+%22callback_data%22%3A+%229%22%7D%2C+%7B%22text%22%3A+%22%2A%22%2C+%22callback_data%22%3A+%22%2A%22%7D%5D%2C+%5B%7B%22text%22%3A+%224%22%2C+%22callback_data%22%3A+%224%22%7D%2C+%7B%22text%22%3A+%225%22%2C+%22callback_data%22%3A+%225%22%7D%2C+%7B%22text%22%3A+%226%22%2C+%22callback_data%22%3A+%226%22%7D%2C+%7B%22text%22%3A+%22-%22%2C+%22callback_data%22%3A+%22-%22%7D%5D%2C+%5B%7B%22text%22%3A+%221%22%2C+%22callback_data%22%3A+%221%22%7D%2C+%7B%22text%22%3A+%222%22%2C+%22callback_data%22%3A+%222%22%7D%2C+%7B%22text%22%3A+%223%22%2C+%22callback_data%22%3A+%223%22%7D%2C+%7B%22text%22%3A+%22%2B%22%2C+%22callback_data%22%3A+%22%2B%22%7D%5D%2C+%5B%7B%22text%22%3A+%22%22%2C+%22callback_data%22%3A+%22no%22%7D%2C+%7B%22text%22%3A+%220%22%2C+%22callback_data%22%3A+%220%22%7D%2C+%7B%22text%22%3A+%22%2C%22%2C+%22callback_data%22%3A+%22.%22%7D%2C+%7B%22text%22%3A+%22%3D%22%2C+%22callback_data%22%3A+%22%3D%22%7D%5D%5D%7D HTTP/1.1" 200 936
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394661&timeout=20 HTTP/1.1" 200 1190
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "POST /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/editMessageText?text=6-3&chat_id=68978487&message_id=17&reply_markup=%7B%22inline_keyboard%22%3A+%5B%5B%7B%22text%22%3A+%22%22%2C+%22callback_data%22%3A+%22no%22%7D%2C+%7B%22text%22%3A+%22C%22%2C+%22callback_data%22%3A+%22C%22%7D%2C+%7B%22text%22%3A+%22%3C%3D%22%2C+%22callback_data%22%3A+%22%3C%3D%22%7D%2C+%7B%22text%22%3A+%22%2F%22%2C+%22callback_data%22%3A+%22%2F%22%7D%5D%2C+%5B%7B%22text%22%3A+%227%22%2C+%22callback_data%22%3A+%227%22%7D%2C+%7B%22text%22%3A+%228%22%2C+%22callback_data%22%3A+%228%22%7D%2C+%7B%22text%22%3A+%229%22%2C+%22callback_data%22%3A+%229%22%7D%2C+%7B%22text%22%3A+%22%2A%22%2C+%22callback_data%22%3A+%22%2A%22%7D%5D%2C+%5B%7B%22text%22%3A+%224%22%2C+%22callback_data%22%3A+%224%22%7D%2C+%7B%22text%22%3A+%225%22%2C+%22callback_data%22%3A+%225%22%7D%2C+%7B%22text%22%3A+%226%22%2C+%22callback_data%22%3A+%226%22%7D%2C+%7B%22text%22%3A+%22-%22%2C+%22callback_data%22%3A+%22-%22%7D%5D%2C+%5B%7B%22text%22%3A+%221%22%2C+%22callback_data%22%3A+%221%22%7D%2C+%7B%22text%22%3A+%222%22%2C+%22callback_data%22%3A+%222%22%7D%2C+%7B%22text%22%3A+%223%22%2C+%22callback_data%22%3A+%223%22%7D%2C+%7B%22text%22%3A+%22%2B%22%2C+%22callback_data%22%3A+%22%2B%22%7D%5D%2C+%5B%7B%22text%22%3A+%22%22%2C+%22callback_data%22%3A+%22no%22%7D%2C+%7B%22text%22%3A+%220%22%2C+%22callback_data%22%3A+%220%22%7D%2C+%7B%22text%22%3A+%22%2C%22%2C+%22callback_data%22%3A+%22.%22%7D%2C+%7B%22text%22%3A+%22%3D%22%2C+%22callback_data%22%3A+%22%3D%22%7D%5D%5D%7D HTTP/1.1" 200 937
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394662&timeout=20 HTTP/1.1" 200 1191
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "POST /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/editMessageText?text=3&chat_id=68978487&message_id=17&reply_markup=%7B%22inline_keyboard%22%3A+%5B%5B%7B%22text%22%3A+%22%22%2C+%22callback_data%22%3A+%22no%22%7D%2C+%7B%22text%22%3A+%22C%22%2C+%22callback_data%22%3A+%22C%22%7D%2C+%7B%22text%22%3A+%22%3C%3D%22%2C+%22callback_data%22%3A+%22%3C%3D%22%7D%2C+%7B%22text%22%3A+%22%2F%22%2C+%22callback_data%22%3A+%22%2F%22%7D%5D%2C+%5B%7B%22text%22%3A+%227%22%2C+%22callback_data%22%3A+%227%22%7D%2C+%7B%22text%22%3A+%228%22%2C+%22callback_data%22%3A+%228%22%7D%2C+%7B%22text%22%3A+%229%22%2C+%22callback_data%22%3A+%229%22%7D%2C+%7B%22text%22%3A+%22%2A%22%2C+%22callback_data%22%3A+%22%2A%22%7D%5D%2C+%5B%7B%22text%22%3A+%224%22%2C+%22callback_data%22%3A+%224%22%7D%2C+%7B%22text%22%3A+%225%22%2C+%22callback_data%22%3A+%225%22%7D%2C+%7B%22text%22%3A+%226%22%2C+%22callback_data%22%3A+%226%22%7D%2C+%7B%22text%22%3A+%22-%22%2C+%22callback_data%22%3A+%22-%22%7D%5D%2C+%5B%7B%22text%22%3A+%221%22%2C+%22callback_data%22%3A+%221%22%7D%2C+%7B%22text%22%3A+%222%22%2C+%22callback_data%22%3A+%222%22%7D%2C+%7B%22text%22%3A+%223%22%2C+%22callback_data%22%3A+%223%22%7D%2C+%7B%22text%22%3A+%22%2B%22%2C+%22callback_data%22%3A+%22%2B%22%7D%5D%2C+%5B%7B%22text%22%3A+%22%22%2C+%22callback_data%22%3A+%22no%22%7D%2C+%7B%22text%22%3A+%220%22%2C+%22callback_data%22%3A+%220%22%7D%2C+%7B%22text%22%3A+%22%2C%22%2C+%22callback_data%22%3A+%22.%22%7D%2C+%7B%22text%22%3A+%22%3D%22%2C+%22callback_data%22%3A+%22%3D%22%7D%5D%5D%7D HTTP/1.1" 200 935
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
DEBUG:urllib3.connectionpool:https://api.telegram.org:443 "GET /bot5600774130:AAFgX-Q5vQ2RCEIsh908iXDy9liuUr1VR84/getUpdates?offset=22394663&timeout=20 HTTP/1.1" 200 23
ERROR:TeleBot:Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
ERROR:TeleBot:Exception traceback:
Traceback (most recent call last):
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 1077, in __threaded_polling
    polling_thread.raise_exceptions()
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\util.py", line 116, in raise_exceptions
    raise self.exception_info
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\util.py", line 98, in run
    task(*args, **kwargs)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 653, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 627, in get_updates
    json_updates = apihelper.get_updates(self.token, offset, limit, timeout, allowed_updates, long_polling_timeout)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 334, in get_updates
    return _make_request(token, method_url, params=payload)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 162, in _make_request
    json_result = _check_result(method_name, result)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 189, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

ERROR:TeleBot:Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
ERROR:TeleBot:Exception traceback:
Traceback (most recent call last):
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 1077, in __threaded_polling
    polling_thread.raise_exceptions()
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\util.py", line 116, in raise_exceptions
    raise self.exception_info
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\util.py", line 98, in run
    task(*args, **kwargs)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 653, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 627, in get_updates
    json_updates = apihelper.get_updates(self.token, offset, limit, timeout, allowed_updates, long_polling_timeout)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 334, in get_updates
    return _make_request(token, method_url, params=payload)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 162, in _make_request
    json_result = _check_result(method_name, result)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 189, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

ERROR:TeleBot:Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
ERROR:TeleBot:Exception traceback:
Traceback (most recent call last):
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 1077, in __threaded_polling
    polling_thread.raise_exceptions()
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\util.py", line 116, in raise_exceptions
    raise self.exception_info
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\util.py", line 98, in run
    task(*args, **kwargs)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 653, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 627, in get_updates
    json_updates = apihelper.get_updates(self.token, offset, limit, timeout, allowed_updates, long_polling_timeout)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 334, in get_updates
    return _make_request(token, method_url, params=payload)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 162, in _make_request
    json_result = _check_result(method_name, result)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 189, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

ERROR:TeleBot:Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
ERROR:TeleBot:Exception traceback:
Traceback (most recent call last):
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 1077, in __threaded_polling
    polling_thread.raise_exceptions()
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\util.py", line 116, in raise_exceptions
    raise self.exception_info
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\util.py", line 98, in run
    task(*args, **kwargs)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 653, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 627, in get_updates
    json_updates = apihelper.get_updates(self.token, offset, limit, timeout, allowed_updates, long_polling_timeout)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 334, in get_updates
    return _make_request(token, method_url, params=payload)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 162, in _make_request
    json_result = _check_result(method_name, result)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 189, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

ERROR:TeleBot:Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
ERROR:TeleBot:Exception traceback:
Traceback (most recent call last):
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 1077, in __threaded_polling
    polling_thread.raise_exceptions()
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\util.py", line 116, in raise_exceptions
    raise self.exception_info
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\util.py", line 98, in run
    task(*args, **kwargs)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 653, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 627, in get_updates
    json_updates = apihelper.get_updates(self.token, offset, limit, timeout, allowed_updates, long_polling_timeout)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 334, in get_updates
    return _make_request(token, method_url, params=payload)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 162, in _make_request
    json_result = _check_result(method_name, result)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 189, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

ERROR:TeleBot:Threaded polling exception: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
ERROR:TeleBot:Exception traceback:
Traceback (most recent call last):
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 1077, in __threaded_polling
    polling_thread.raise_exceptions()
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\util.py", line 116, in raise_exceptions
    raise self.exception_info
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\util.py", line 98, in run
    task(*args, **kwargs)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 653, in __retrieve_updates
    updates = self.get_updates(offset=(self.last_update_id + 1),
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\__init__.py", line 627, in get_updates
    json_updates = apihelper.get_updates(self.token, offset, limit, timeout, allowed_updates, long_polling_timeout)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 334, in get_updates
    return _make_request(token, method_url, params=payload)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 162, in _make_request
    json_result = _check_result(method_name, result)
  File "C:\Users\brusl\AppData\Local\Programs\Python\Python310\lib\site-packages\telebot\apihelper.py", line 189, in _check_result
    raise ApiTelegramException(method_name, result, result_json)
telebot.apihelper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 409. Description: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running

